courses/api.py

```diff
@@ -1403,25 +1432,34 @@ def get_verifiable_credentials_payload(certificate: BaseCertificate) -> dict:
                 "id": url or f"urn:uuid:{certificate.uuid}",
                 "achievementType": achievement_type,
                 "type": ["Achievement"],
-                "image": {
-                    "id": "https://github.com/digitalcredentials/test-files/assets/206059/01eca9f5-a508-40ac-9dd5-c12d11308894",
-                    "type": "Image",
-                    "caption": "MIT Learn Certificate logo",
-                },
                 "criteria": {
-                    # TODO: Need to figure out what this needs to be
+                    # TODO: Need to figure out what this needs to be #noqa: TD002, TD003, FIX002
                     "narrative": "If you wanted to add some kind of criteria, e.g. a list of courses or modules, etc. CAN BE MARKDOWN"
```

Comment:
Ok, this is a bit complex. 

For programs, let's make this a markdown list of the courses that make up the program. It would be nice to make these links too, but I don't think we can do that for MVP. 

For courses, I'm not sure what to put in here. I might regret this later, but let's use the `what_you_learn` section of the CMS course page. If it's empty (which it will be in many cases), I guess we have to raise an exception because this isn't a nullable field in the credential. I will commit to looking for those errors and filling in these fields.